content/acra/configuring-maintaining/general-configuration/acra-keys/list.md

```diff
@@ -107,9 +107,10 @@ $ acra-keys list
 
 INFO[0000] Initializing ACRA_MASTER_KEY loader...       
 INFO[0000] Initialized default env ACRA_MASTER_KEY loader 
-Key purpose                  | Client/Zone ID | Key ID
------------------------------+----------------+-------------------------
-audit log signature key      |                | audit-log
-client storage symmetric key | user1          | client/user1/storage-sym
+Key purpose     | Client/Zone ID | Key ID
+----------------+----------------+------------------
+audit_log       | secure         | secure_log_key
```

Comment:
what means `secure` and where user should get it?

Reply:
I have no clue why there is a `secure` and why `sym_key` is printed twice :) I just run the examples to be sure they work and copy the output

Reply:
hm... it is first part of filename: `secure_log_key`... - https://github.com/cossacklabs/acra/blob/master/keystore/filesystem/server_keystore.go#L927

but I don't know why we print same type of keys twice...
Anyway it is wrong output. I suggest to write in docs correct and expected output and create new task for futher investigation why it works so and fix. What do you think?

Reply:
Ok. But since I don't know how real records should look like, I would keep everything as it was